packages/flutter/test/material/tabs_test.dart

```diff
@@ -1758,6 +1758,68 @@ void main() {
     semantics.dispose();
   });
 
+  testWidgets('can be notified of TabBar onTap behavior', (WidgetTester tester) async {
+    Widget buildFrame({
+      TabController controller,
+      List<String> tabs,
+      String value,
+      bool isScrollable = false,
+      Color indicatorColor,
+    }) {
+      return boilerplate(
+        child: Container(
+          child: TabBar(
+            controller: controller,
+            tabs: tabs.map<Widget>((String tab) => Tab(text: tab)).toList(),
+              onTap: (int index) {
+                if (controller.index != index) {
```

Comment:
This is what we don't want developers to do; onTap is a "me too" notification. 

Maybe this is leftover from a previous version of the PR?